docs/source/posts/2022/2022-07-25-week-6-mohamed.rst

```diff
@@ -0,0 +1,39 @@
+Week 6: Fixing the ``Timeline`` issues and equipping it with more features
+==========================================================================
+
+.. post:: July 25 2022
+   :author: Mohamed Abouagour
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- Improved the ``PlaybackPanel`` by adding speed control and the ability to loop the animation. Also, fixed the lagging issue of play and pause buttons and composed them into a single play/pause button.
+
+- Updated the old tutorials' syntax to match the other tutorials and added a new tutorial on position animation using spline interpolation. And added unit tests for the ``PlaybackPanel`` and the newly added color converters in ``colormap.py``.
```

Comment:
I think these lines should be something like this
`... spline interpolation and added unit tests ...`
or
`... spline interpolation. Added unit tests ...`